templates/discourse/files/.platform/applications.yaml

```diff
@@ -0,0 +1,143 @@
+- name: app
+  disk: 1024
+  type: "ruby:2.6"
+  variables:
+      env:
+          NVM_VERSION: "v0.35.3"
+          NODE_VERSION: "12.16.3"
+  size: M
+  resources:
+     base_memory: 1001 # between 64 and 1024 max
+     memory_ratio: 512 # integer between 128 and 1024 max
+  hooks:
+      build: |
+          # move into place our config. We are not using the platform_sh_rails helper GEM
+          mv config/discourse.platformsh.conf config/discourse.conf
+          # move into place required binaries
+          mv public/ _public/
+          # create symbolic link form app/tmp to /tmp so Bootsnap doesn't die.
+          ln -s /tmp tmp
+          # install nvm
+
+          unset NPM_CONFIG_PREFIX
+          curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.35.2/install.sh | dash
+          export NVM_DIR="$PLATFORM_APP_DIR/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+          nvm current
+          nvm install 12.16.3
```

Comment:
@todo Regeneralize this.

Reply:
It is better to use "nvm install stable" or something like this. Look at my playbook repo for a much better take on installing NVM (that enables caching the nvm downloads).

Reply:
Wouldn't `stable` mean it changes when upstream changes what it considers stable?